metaflow/plugins/kfp/tests/flows/resources_flow.py

```diff
@@ -126,6 +126,7 @@ class ResourcesFlow(FlowSpec):
     @resources(
         local_storage="242",
         cpu="0.6",
+        local_storage="242",
```

Comment:
Looks like this is a typo?

Reply:
Ah not sure where that comes from. Thanks for spotting it!